vgrep.go

```diff
@@ -122,30 +122,29 @@ func main() {
 			fmt.Fprintf(os.Stderr, "error loading cache: %v\n", err)
 			os.Exit(1)
 		}
+
+		// Execute the specified command and/or jump directly into the
+		// interactive shell.
+		if len(v.matches) > 0 {
+			v.commandParse()
+			v.waiter.Wait()
+		}
+
+		os.Exit(0)
```

Comment:
I'm now wondering if it wouldn't be better to also exit with 1 if a command has no matches to operate on.

```
vgrep "matches zero lines"  # exits with 1
vgrep --show 3              # exit with 0?
```